include/compare_main.h

```diff
@@ -41,7 +41,11 @@ struct CompareOptions {
     bool output_vcf { false };
     bool illumina { false };
     bool clean { false };
+    float min_absolute_gene_coverage { 3.0 };
```

Comment:
what is absolute gene coverage? number of reads on the gene, or the mean depth?

Reply:
gene coverage is always mean depth. Then we have minimum absolute gene coverage, a flat value that the mean gene coverage needs to be kept, and the min relative gene coverage, a proportion WRT the global coverage